homeassistant/components/evohome/__init__.py

```diff
@@ -45,6 +49,7 @@
             {
                 vol.Required(CONF_USERNAME): cv.string,
                 vol.Required(CONF_PASSWORD): cv.string,
+                vol.Optional(CONF_HIGH_PRECISION, default=False): bool,
```

Comment:
Why do we need an option? Can't we just pick a sane precision level?

Reply:
Your point is valid - the option was a left over from the way things were done when it was a custom component, and other issues (e.g. API rate limits) had a part to play.

I can just take it out.